libs/libc/time/lib_strftime.c

```diff
@@ -368,6 +368,15 @@ size_t strftime(FAR char *s, size_t max, FAR const char *format,
              }
              break;
 
+            /* %R: Shortcut for %H:%M. */
+
+           case 'R':
+             {
+               len = snprintf(dest, chleft, "%02d:%02d",
+                tm->tm_hour, tm->tm_min);
```

Comment:
```suggestion
               len = snprintf(dest, chleft, "%02d:%02d",
                              tm->tm_hour, tm->tm_min);
```

Reply:
Thanks, I have just rebased the branch according to your suggestions.